pay-id-api-spec/pay-id.v1.yml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.0
+info:
+  title: PayID
+  version: '0.1'
+servers:
+  - url: 'https://'
+# TODO(keefertaylor): Headers can be added in this file or manually set on the client. Evaluate and implement headers.
+paths:
+  '/{path}':
+    parameters:
+      - schema:
+          type: string
+        name: path
+        in: path
+        required: true
+    get:
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  address:
+                    type: string
```

Comment:
Might be worth pulling this schema out into a named schema?

Here's a named schema with examples I wrote for the Github Payout spec:

```
components:
  schemas:
    Repository:
      type: object
      required:
        - name
      properties:
        id:
          type: string
          format: uuid
          example: acd48109-dbf5-4436-8f84-56d8686533a4
        organization_id:
          type: string
          format: uuid
          example: d290f1ee-6c54-4b01-90e6-d701748f0851
        name:
          type: string
          example: 'xpring-common-js'
```

Reply:
Also, the return payload also has a `network` property, which will match the Accept header.

So:
`application/xrpl-testnet+json` =>
```
{
  "network": "xrpl-testnet",
  "address": "T-Address",
}
```

Reply:
> Might be worth pulling this schema out into a named schema?

Done.

> Also, the return payload also has a network property, which will match the Accept header.

Noted. My goal here is to keep things compatible to make sure this change is a zero logic change. A future change will implement network / move to the proper PayID service.